ros/api/v0/recommendation.py

```diff
@@ -0,0 +1,62 @@
+from ros.lib.models import Rule, RhAccount, System, db
+from ros.lib.utils import is_valid_uuid, identity
+from flask_restful import Resource, abort, fields, marshal_with
+from flask import request
+
+
+class RecommendationsApi(Resource):
+
+    recommendation_fields = {
+        'rule_id':  fields.String,
+        'description': fields.String,
+        'reason': fields.String,
+        'resolution': fields.String,
+        'condition': fields.String
+    }
+    data_fields = {
+        'host_id': fields.String,
+        'number_of_recommendations': fields.Integer,
+        'recommendations': fields.List(fields.Nested(recommendation_fields))
+    }
+
+    @marshal_with(data_fields)
+    def get(self, host_id):
+        if not is_valid_uuid(host_id):
+            abort(404, message='Invalid host_id, Id should be in form of UUID4')
+
+        ident = identity(request)['identity']
+
+        filter_description = request.args.get('description')
+
+        account_query = db.session.query(RhAccount.id).filter(RhAccount.account == ident['account_number']).subquery()
+        system = db.session.query(System) \
+            .filter(System.account_id.in_(account_query)).filter(System.inventory_id == host_id).first()
+
+        if not system:
+            abort(404, message="host with id {} doesn't exist"
+                  .format(host_id))
+        rule_hits = system.rule_hit_details
+        recommendations = []
```

Comment:
Can we rename the following in order to improve readability [here](https://github.com/RedHatInsights/ros-backend/pull/64/files#diff-95c96a33a2f02363efd7c611c4c1f95b865513fb21a368b0c7e6a2ba69060ffcR53)?
```suggestion
        recommendations_list = []
```

Reply:
Done